### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by uwinema**

### CODE REVIEW

**Review by Fbehr-data — APPROVED** [review]
looks good

File: README.md [hunk 1]
```diff
@@ -136,5 +136,20 @@ to
 |1        | 0.4        | 0.7         |
 
 ### Train-Test-Split
+Splitting our dataset is essential for an unbiased evaluation of prediction performance.we split randomly our dataset into 3 subset:
+- Training set : is applied to train, or fit, the model
+- Validation set :is used for unbiased model evaluation during hyperparameter tuning
+- Test set:is needed for an unbiased evaluation of the final model.
```
Comment (langer-net) [inline README.md:142]: Training set: is applied ...
Validation set: is used for ...
Test set: is needed ...
-> just check the whitespaces

File: README.md [hunk 2]
```diff
@@ -136,5 +136,20 @@ to
 |1        | 0.4        | 0.7         |
 
 ### Train-Test-Split
+Splitting our dataset is essential for an unbiased evaluation of prediction performance.we split randomly our dataset into 3 subset:
```
Comment (langer-net) [inline README.md:139]: Line 139:  ... performance. We randomly split our dataset into 3 subsets:

File: README.md [hunk 3]
```diff
@@ -136,5 +136,20 @@ to
...
+- Training set : is applied to train, or fit, the model
+- Validation set :is used for unbiased model evaluation during hyperparameter tuning
+- Test set:is needed for an unbiased evaluation of the final model.
+Now, in order to do that we need first to import the train_test_split function that we created ,like this:
```
Comment (langer-net) [inline README.md:143]: ... that we created, like this ...

-> whitespaces on the comma

File: README.md [hunk 4]
```diff
@@ -136,5 +136,20 @@ to
...
+```BASH
+from train_test_function import train_test_split_fields
+```
+the split is done by that function ,we only set the train_size as the test_size will adjust accordingly. we also set the random_state to 42.
```
Comment (langer-net) [inline README.md:147]: ... that function, we only ...

... accordingly. The random_state was set to 42.

-> whitespaces on the comma
